classes/01class/exercises/c01-git01/mail2kvanitha/README.md

```diff
@@ -0,0 +1,33 @@
+# c01-git01
+
+## Commands executed
+
+1. Commands typed to solve the "Merging" exercise
+git checkout master
+git merge <branch-name>
+git checkout <branch-name>
+git merge master
+
+```
+
+2. Commands typed to solve the "Rebasing" exercise
+git checkout -b branch-name
+git commit
+git checkout master
+git commit
+git checkout <branch_name>
+git rebase master 
+
+```
+
+3. Commands typed to solve the "Revert and reset" exercise
+git reset master
+git checkout pushed
+git revert pushed
```

Comment:
adjust formatting

Reply:
Adjusted formatting, Please check and confirm if anything pending.